### DISCUSSION TITLE
php artisan commands returning output by attaching dot (.) symbol in first line

### DISCUSSION THREAD

**Discussion by prashant-malla** [laravel/framework#41547] [General]
When I run an artisan command like `php artisan --version`, It gives output by attaching '.'(dot) in the first line like `.Laravel Framework 9.5.1`

Due to this in each js file, there is '.' in the first line which gives an error.

![problem artisan](https://user-images.githubusercontent.com/38640420/158927411-0bd30be1-d5b4-46f0-b882-4bb3d2fd051d.JPG)

**Comment by shehrozafzal** [comment]
Check complete laravel request lifecycle's files one by one and look for extra dot. i faced this problem once and i found .(dot) in web.php 
best of luck.

edited: in command only case check console.php too
